nomad/poc-2.hcl

```diff
@@ -1,21 +1,37 @@
 /*
-    poc-2: distributed holochain gossip test
+    poc-2: run a cargo test that has a server and a client role
 
-    - public reachable node runs the bootstrap and turn servers
-    - nat'ed conductors are configured to use these public services
-    - a simple happ generates data and ensures it is getting data back from others
+    - a public reachable node runs the server role
+    - defined number of nat'ed nodes run the client roles, getting the server's IP and port via environment variables
+
+    - improvement TODOs:
+        - add prestart tasks that only compile the binaries
+            - server
+            - clients
+        - start run the clients when the server is ready
 */
 
+variables {
+    GIT_URL = "https://github.com/steveej-forks/holochain.git"
```

Comment:
Should this point back to holochain/holochain now it's being merged? Is there something in the fork which needs doing upstream?